harvest/plugin/dolt_options_plugin.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd 
```

Comment:
A user will be writing similar plugins to this one so let me know if this is too obtuse or make it hard for a user to create their own plugin.

Reply:
Looks simple and intuitive to me :)